tests/data/bundle.yaml

```diff
@@ -10,46 +10,27 @@ machines:
 services:
   containerd:
     charm: cs:~containers/containerd
-    resources: {}
   easyrsa:
     charm: cs:~containers/easyrsa
     num_units: 1
-    resources:
-      easyrsa: 5
     to:
     - '1'
   etcd:
     charm: cs:~containers/etcd
     num_units: 1
     options:
       channel: 3.4/stable
-    resources:
-      core: 0
-      etcd: 3
-      snapshot: 0
     to:
     - '0'
   flannel:
     charm: cs:~containers/flannel
-    resources:
-      flannel-amd64: 653
-      flannel-arm64: 650
-      flannel-s390x: 637
   kubernetes-master:
-    charm: {{master_charm}}
+    charm: cs:~containers/kubernetes-master
```

Comment:
What happened to `{{master_charm}}`? I think the integration tests need that to deploy a locally built master instead of pulling it from the charm store: https://github.com/charmed-kubernetes/charm-kubernetes-master/blob/91e7ae2543db2629504ab6efd9ecac8687396fa0/tests/integration/test_kubernetes_master_integration.py#L11-L13

Reply:
gah!  they absolutely do.  i modified that for a local deployment to make sure i could just drop all resources from the test bundle.yaml.  forgot to put it back :/